### PR TITLE
Akanksha bootcamp details

### PR DESCRIPTION
**Pull request by AkankshShigwan**
working on this @prasadhonrao please review this -> I am still working on this it's not the final code

### CODE REVIEW

File: ui/src/components/courses.jsx
Comment (prasadhonrao) [inline ui/src/components/courses.jsx:1]: Pls correct file name casing

File: ui/src/components/courses.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import Table from 'react-bootstrap/Table';
```
Comment (prasadhonrao) [inline ui/src/components/courses.jsx:1]: I think you can define a Course componene for this.

File: ui/src/index.css [hunk 2]
```diff
@@ -37,3 +37,7 @@ img {
   width: 100%;
   height: 100%;
 }
+
+.card-head-color{
+  background-color: #e05433 !important;
```
Comment (prasadhonrao) [inline ui/src/index.css:42]: Avoid using !important as it gets difficult to track the CSS hierarchy

File: ui/src/pages/bootcamps/ManageCoursesPage.jsx [hunk 3]
```diff
@@ -1,5 +1,61 @@
...
 const ManageCoursesPage = () => {
-  return <div>Manage Courses Page</div>;
+
+  //redirect to courses page (need to implement in future)
+  function redirect_addCourses(e) {
+    window.location.href = `/bootcamps/:bootcampId/courses/add`;
+  }
+
+  return (
+    <>
```
Comment (prasadhonrao) [inline ui/src/pages/bootcamps/ManageCoursesPage.jsx:14]: When you are returning a section, fragment i.e. <> is not required

**Review by prasadhonrao — CHANGES_REQUESTED** [review]

File: ui/src/components/courses.jsx
Comment (AkankshShigwan) [reply ui/src/components/courses.jsx:1]: done

File: ui/src/components/courses.jsx (hunk 1 above)
Comment (AkankshShigwan) [reply ui/src/components/courses.jsx:1]: It is previous code which is not merge with development branch that's why you can see this changes. In bootcamp details page not using courses data or structure.

File: ui/src/index.css (hunk 2 above)
Comment (AkankshShigwan) [reply ui/src/index.css:42]: noted

File: ui/src/pages/bootcamps/ManageCoursesPage.jsx (hunk 3 above)
Comment (AkankshShigwan) [reply ui/src/pages/bootcamps/ManageCoursesPage.jsx:14]: noted

**Comment by AkankshShigwan** [comment]
@prasadhonrao please review this and also, I have added comments for the changes

**Pull request merged**